tests/rest/admin/test_user.py

```diff
@@ -879,6 +879,44 @@ def test_order_by(self) -> None:
         self._order_test([self.admin_user, user1, user2], "creation_ts", "f")
         self._order_test([user2, user1, self.admin_user], "creation_ts", "b")
 
+    def test_filter_admins(self) -> None:
+        """
+        Tests whether the various values of the query parameter admins lead to the
+        expected result set.
+        """
+
+        # Register an additional non admin user
+        self.register_user("user", "pass", admin=False)
+
+        # Query all users
+        channel = self.make_request(
+            "GET",
+            f"{self.url}",
+            access_token=self.admin_user_tok,
+        )
+        self.assertEqual(200, channel.code, channel.result)
+        self.assertEqual(2, channel.json_body["total"])
```

Comment:
It might be a little clearer to make the test check that the usernames of the users we returned are the users we expect (i.e. whatever the admin user's name is is, and `user`). But if this is fiddly to do or not what the existing tests do, this is probably good enough.

Reply:
Okay, then I'll leave it that way.